.docker/install-plugin-theme.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Install plugins
+echo "Installing plugins"
+wp --allow-root --url=wordcamp.test plugin install akismet buddypress bbpress jetpack wp-multibyte-patch wordpress-importer polldaddy liveblog wp-super-cache custom-content-width camptix-network-tools email-post-changes tagregator supportflow camptix-pagseguro camptix-payfast-gateway camptix-trustpay camptix-trustcard camptix-mercadopago camptix-kdcpay-gateway campt-indian-payment-gateway camptix
+
+# Install themes
+echo "Installing themes"
+wp --allow-root --url=wordcamp.test theme install twentyten twentyeleven twentytwelve twentythirteen twentyfourteen
```

Comment:
`MUST` - I think we'll need the rest of the Twenty* themes too. We can't rely on Core to bundle them in the future, because that changes over time.

Reply:
This would add some non-neglible time to setup procedure. I added this because they were also getting added by meta environment, and TBH I wanted to install not more than 2 3 themes.

User can always add whatever themes they like on their own. What do you think?

Reply:
I think the fundamental principle of development environments is that they should match production as closely as possible. Having a fast setup time is nice, but I think accurately representing production is far more important.

Reply:
Added themes in https://github.com/WordPress/wordcamp.org/pull/113/commits/db214d69ebe62cfa4be3af4e8432092c90f72061